tests/matrix_free/stokes_computation.with_mpi=true.with_p4est=true.mpirun=4.output

```diff
@@ -0,0 +1,15 @@
+
+DEAL:2d::Sinker problem in 2D.
+DEAL:2d::n_sinker: 4       max/min viscosity ratio: 1000.00
+DEAL:2d::
+DEAL:2d::Number of active cells: 1024 (on 6 levels)
+DEAL:2d::Number of degrees of freedom: 9539 (8450+1089)
+DEAL:2d::Solved-in 62 iterations, final residual: 5.37736e-08
```

Comment:
Is this number of iterations expected? I guess it is pretty high because of the viscosity contrast, but it would be good to get this verified. Could you maybe run the 2D test with one more refinement level just to see how it scales?

Reply:
One more refinement level is 61 iterations and one more refinement after that is 61 iterations as well.

Reply:
Do you think it would make sense to add that second refinement? The numbers look nice, but I guess it is much more intuitive in a future change if one sees immediately from the expected program output that we expect constant iteration counts. 38k DoFs should still be fine for the tester, given that it is a pretty good solver.

Reply:
I would keep one 3D solve, yes.